lib/components/MenuRenderer/testHelper.tsx

```diff
@@ -1,5 +1,6 @@
 import '@testing-library/jest-dom/extend-expect';
-import React, { useState } from 'react';
+import { useState } from 'react';
+import * as React from 'react';
```

Comment:
Maybe destructure `FunctionComponent` so we dont have to have this separate import